src/Host/Client/Impl/rtvs/R/overrides.R

```diff
@@ -55,8 +55,9 @@ show_file <- function(files, header, title, delete.file) {
 }
 
 install.packages <- function(...) {
+    invisible(rtvs:::send_notification('!BeforePackagesInstalled'))
```

Comment:
Now that the callback is wired up async, this should be `send_request_and_wait_response`.